map/section-1.md

```diff
@@ -20,4 +20,13 @@ We invite and encourage you to add any company, event or initiative that helps o
 6. You can add as many additional tags as you wish, but make sure that the default tags are preserved.
 
 ## About the map
-Our map is part of [mapoftomorrow.org](http://mapoftomorrow.org). On their website you can find a lot of interesting green companies & initiatives that are not related to film production but may still be interesting to you! If you stumble upon anything that is relevant to our industry but is not shown on our page please add the ***#greenfilm*** tag and the appropriate category tags to make it appear!
\ No newline at end of file
+Our map is part of [mapoftomorrow.org](http://mapoftomorrow.org). The great thing about it: It is public, free and everything is open source. On their website you can find a lot of interesting green companies & initiatives that are not related to film production but may still be interesting to you! If you stumble upon anything that is relevant to our industry but is not shown on our page please add the ***#greenfilm*** tag and the appropriate category tags to make it appear!
```

Comment:
> The great thing about it: It is public, free and everything is open source.

I would remove that part, does not feels relevant. People can directly use it so its obvious that it is free and open? Also the fact that the map is open source is not really interesting to most end users I guess. Maybe we can add a link to their GitHub somewhere but I would not place it that prominent.

Reply:
I somehow would like to point out that it is and will stay public and free. But I think you're right about the open source notice.